libs/libarchfpga/src/arch_util.cpp

```diff
@@ -616,6 +642,35 @@ void alloc_and_load_default_child_for_pb_type(t_pb_type* pb_type,
             copy->ports[i].port_power->buffer_type = POWER_BUFFER_TYPE_NONE;
         }
     }
+    // Special case for latch - fill secondary data fields
+    if (pb_type->class_type == LATCH_CLASS) {
+        copy->num_ports_sec = pb_type->num_ports;
+        copy->ports_sec = (t_port*)vtr::calloc(pb_type->num_ports, sizeof(t_port));
```

Comment:
We're trying to get away from vtr::calloc so we can convert everything to new/delete.  So I'd convert to new/delete if possible, and malloc if not (as it will be easier to convert later than calloc).  It seems you probably don't need calloc anyway as it looks like you're explicitly setting everything.

Reply:
Changed that to `new`